scheds/rust/scx_mitosis/src/bpf/mitosis.bpf.c

```diff
@@ -1305,6 +1372,68 @@ void BPF_STRUCT_OPS(mitosis_cgroup_move, struct task_struct *p,
 	update_task_cell(p, tctx, to);
 }
 
+/*
+ * Tracepoint fallbacks - only active when CPU controller is disabled.
+ * These provide cgroup tracking when SCX cgroup callbacks don't fire.
+ */
+SEC("tp_btf/cgroup_attach_task")
+int BPF_PROG(tp_cgroup_attach_task, struct cgroup *cgrp, const char *cgrp_path,
+	     struct task_struct *task, bool threadgroup)
+{
+	int ret;
+
+	if (!cpu_controller_disabled)
+		return 0;
+
+	/* Initialize cgrp_ctx if needed (first task attached to this cgroup) */
+	ret = tp_init_cgrp_ctx(cgrp);
+	if (ret) {
+		scx_bpf_error(
+			"tp_cgroup_attach_task: tp_init_cgrp_ctx failed for cgid %llu: %d",
+			cgrp->kn->id, ret);
+		return 0;
+	}
+
+	/*
+	 * Bump configuration_seq to trigger task cell refresh via maybe_refresh_cell()
+	 * on the next scheduling event. We can't call update_task_cell() directly
+	 * because tracepoints have read-only access to task arguments.
+	 */
+	__atomic_add_fetch(&configuration_seq, 1, __ATOMIC_RELEASE);
```

Comment:
Why do we need to trigger task cell refresh here? init_cgrp_ctx now bumps if it needs to create a new cell, otherwise there shouldn't be any need to trigger the timer.

Reply:
This is in leiu of update_task_cell in cgroup_move. I think this fixed affinity violations going high when churning through containers but will confirm.